### PR TITLE
Addition of Attacks/Reports

### PR DESCRIPTION
**Pull request by mabdullah22**
I have added attacks/reports column and added some reports. This can be expanded further and more reports/attacks can be added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,8 +79,8 @@ Important Note: This standard for crypto wallets security is product of ongoing
 
 ### 6. Communication
 
-|  #  | Description                                                                                                                                                                                                                                                                      | Comment            |
-| --- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+|  #  | Description                                                                                                                                                                                                                                                                      | Comment            | Attacks/Reports |
```
Comment (matiassequeira) [inline README.md:82]: Can you add the attacks/reports column to section #5? I've a report related to 5.2

**Review by matiassequeira — CHANGES_REQUESTED** [review]
Hello @mabdullah22, thank you for your valuable input and congrats on your reports. Once you add the new column I'll merge your PR

**Comment by mabdullah22** [comment]
I have added it. Cheers :)

**Review by matiassequeira — APPROVED** [review]
lgtm

**Pull request merged**